samtranslator/model/events.py

```diff
@@ -1,6 +1,13 @@
+from typing import Any, Dict, List, Optional
+
 from samtranslator.model import GeneratedProperty, Resource
 from samtranslator.model.intrinsics import fnGetAtt, ref
 
+# Event Rule Targets Id and Logical Id has maximum 64 characters limit
+# https://docs.aws.amazon.com/eventbridge/latest/APIReference/API_Target.html
+EVENT_RULE_ID_MAX_LENGTH = 64
```

Comment:
nit

```suggestion
EVENT_RULE_LOGICAL_ID_MAX_LENGTH = 64
```

Reply:
It's not logical ID though 👀

Reply:
It is the logical id for `AWS::Events::Rule` resource.

Reply:
Where does it say that in https://docs.aws.amazon.com/eventbridge/latest/APIReference/API_Target.html?